### PR TITLE
breakdown by pronouns + some styling

### PR DESCRIPTION
**Pull request by richarddushime**
Requesting for a Review 
I added the calculate_breakdown_of_pronouns function and modified dashboard function  and updating the template dashboard.html  with the new table section and some  CSS styles,
Attached is the  screenshot picture of the dashboard since  Student-intern-data directory is nt online 
127-0-0-1-5000-dashboard data

Note i added the statuses but if not needed i would remove the column
if the status of a student is changed it automatically updates the table
Thank you

### CODE REVIEW

**Comment by rowlandm** [comment]
Ok , I need to give you nicer data and then you can test things more easily too.

Great work, I'll check it later tonight or tomorrow

**Comment by richarddushime** [comment]
> Ok , I need to give you nicer data and then you can test things more easily too.
> 
> Great work, I'll check it later tonight or tomorrow

I would also Prefer  that you can take a screenshot of the dashboard on your end so that  i can see if really what i have is what you have to be on same  page , or using the demo data  you can push the folder/directory  of student-intern-data

**Comment by rowlandm** [comment]
> > Ok , I need to give you nicer data and then you can test things more easily too.
> > Great work, I'll check it later tonight or tomorrow
> 
> Thank you I would also Prefer that you can take a screenshot of the dashboard on your end so that i can see if really what i have is what you have to be on same page , or using the demo data you can push the folder/directory of student-intern-data

I can't - the data is private.

But I can improve the public data - [which I have done in the public data repo](https://github.com/WEHI-ResearchComputing/student_intern_data_public).


and the query I am getting using that public data is here:

```
sqlite> select pronouns, count(*) from Students group by pronouns;
he/him|6
she/her|8
they/them|6
```

**Comment by rowlandm** [comment]
This is what the updated public data looks like for the original dashboard view:

**Comment by rowlandm** [comment]
git fetch origin pull/8/head:breakdown_by_pronouns
git checkout breakdown_by_pronouns

https://stackoverflow.com/questions/27567846/how-can-i-check-out-a-github-pull-request-with-git

File: app.py [hunk 1]
```diff
@@ -499,15 +527,15 @@ def dashboard():
     cursor.execute('SELECT * FROM Statuses')
     statuses = cursor.fetchall()
 
-    rows_to_extract = [9, 10,11,12,13,14]
+    rows_to_extract = [9, 10, 11, 12, 13, 14]
```
Comment (rowlandm) [inline app.py:530]: This is my poor code - I am going to fix it after I approve the pull request.

**Review by rowlandm — APPROVED** [review]
Looks great Richard! It tested well with the new data in studnet_intern_data_public.

**Pull request merged**